File: app/lib/models/service.dart
class Service{
  int id;
  String name;
  int cost;
  int premium;
  int active;
  String resume;
  String largeDescription;
  String img;

  Service({ this.id, this.name, this.cost, this.premium, this.active, this.resume, this.largeDescription, this.img });

  factory Service.fromJson(Map<String, dynamic> json){
    return Service(
      id: json['id'],
      name: json['name'],
      cost: json['cost'],
      premium: json['premium'],
      active: json['active'],
      resume: json['resume'],
      largeDescription: json['large_description'],
      img: json['img']
    );
  }
}

File: laravel/app/Models/service.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class service extends Model
{
    use HasFactory;

    protected $table = 'services';

    protected $fillable = [
        'name',
        'cost',
        'premium',
        'resume',
        'large_description',
        'img'
    ];

    protected $attributes = [
        'premium' => 0,
        'img' => null
    ];
}
